assets/js/blocks/cart-checkout/cart/full-cart/cart-line-item-row.js

```diff
@@ -43,6 +44,20 @@ const CartLineItemRow = ( { lineItem = {} } ) => {
 		isPending: itemQuantityDisabled,
 	} = useStoreCartItem( key );
 
+	// Store item quantity in local state so the UI can update independently
+	// of store/server updates.
+	const [ localQuantity, updateLocalQuantity ] = useState( quantity );
```

Comment:
any reason why other components are not using `localQuantity` instead of `quantity` it should make other updates immediate (like saving and item total), the server can later update.

Reply:
Added this as part of the recent changes, definitely makes it better seeing the discount/totals update live.